catalogue_pipeline/recorder/src/test/scala/uk/ac/wellcome/platform/recorder/services/RecorderWorkerServiceTest.scala

```diff
@@ -146,32 +146,30 @@ class RecorderWorkerServiceTest
     fromJson[RecorderWorkEntry](content).get shouldBe RecorderWorkEntry(expectedWork)
   }
 
-  private def withRecorderWorkerService(table: Table, bucket: Bucket, queue: Queue)(
-    testWith: TestWith[RecorderWorkerService, Assertion]) = {
+  private def withRecorderWorkerService[R](table: Table, bucket: Bucket, queue: Queue)(
```

Comment:
It’s unrelated to this PR, but why does this fixture take a Queue *and* create its own? 🤔